mobile/iOS/starterapp/Podfile

```diff
@@ -1,8 +1,7 @@
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
```

Comment:
Per  @AnthonyOliveri, this line is optional.  It does get rid of some annoying warnings so I'm OK keeping it.